src/apps/api/controllers/postcode-to-region-lookup.js

```diff
@@ -0,0 +1,27 @@
+const { getDITRegionFromUKPostcode } = require('../services')
+const { getOptions } = require('../../../lib/options')
+
+async function postcodeToRegionLookupHandler(req, res) {
+  const { token } = req.session
+  const { postcode } = req.params
+
+  try {
+    const { results } = await getDITRegionFromUKPostcode(postcode)
+    if (results.length === 0) {
+      // Unknown postcode.
+      res.json({})
+      return
+    }
+
+    const regionName = results[0].regionName
+    const regions = await getOptions(token, 'uk-region')
+    const region = regions.filter((region) => region.label === regionName)
+    res.json(region[0])
```

Comment:
You could use `Array.find`

```suggestion
    return regions.find((region) => region.label === regionName)
```

Reply:
👍